salt/metalk8s/addons/dex/deployed/login.sls.in

```diff
@@ -0,0 +1,14 @@
+#!jinja | metalk8s_kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
```

Comment:
After #2050 we'll stop using `kubeconfig` and `context` completely, so I don't know how you want to handle that, maybe rebase and remove it right now...

Reply:
We still support it just it's not needed so we can merge like this and remove it after